Documentation/concepts/kubernetes/policy.rst

```diff
@@ -44,6 +44,16 @@ NetworkPolicy
 For more information, see the official `NetworkPolicy documentation
 <https://kubernetes.io/docs/concepts/services-networking/network-policies/>`_.
 
+Known missing features for Kubernetes Network Policy:
+
++-------------------------------+----------------------------------------------+
+| Feature                       | Tracking Issue                               |
++===============================+==============================================+
+| ``ipBlock`` set with a pod IP | https://github.com/cilium/cilium/issues/9209 |
```

Comment:
AFAIK this isn't actually specified in KNP standards.

Reply:
That's true, but it is also not specified anywhere that it can't be a pod IP. There are upstream k8s tests that were currently skipping because of this limitation. https://github.com/cilium/cilium/blob/d80e66fd95fe07976197575abf4e982220e61388/test/kubernetes-test.sh#L76-L83

Reply:
I guess it's fair. Probably worth adding a comment to the related issue though to explain that the same traffic can (and *should*) be allowed via endpointSelector/podSelector policy instead of ipBlock - ie show the mitigation in case someone clicks through to that issue.